projects/module-1/tic-tac-toe-solo-v2.md

```diff
@@ -94,14 +96,14 @@ In a game like Tic Tac Toe, it is tempting to manipulate the DOM first. Remember
 This workflow is not required, but will help you meet the overall requirements of the project.
 
 1. Plan out the HTML layout (colors and icons do not need to match, but overall layout should closely match the demo video)
-2. Create the Player class
-3. Create the Game class
-4. Make game fully playable without the DOM (manually updating the Game.board data, etc, from your console) to force yourself to think data-model-first
-5. Create central game board on the DOM
-6. Connect Game data model to the DOM
-7. Display the Player data in the sidebars
+2. Create the functions that describe/update the players and their data
+3. Create the functions that describe/update the game board and gameplay
+4. Reflect: without thinking about the DOM, could you call functions and make an entire game work? Would your data update properly? Would your game be able to know when someone has won? etc
```

Comment:
Maybe instead of 'make an entire game work', 'have the underlying data of the game be appropriately updated'?

I just worry students won't grasp that 'making the game work' means the logic of the game, and not the view layer.

Reply:
I thought about this too. I'll update the language to be a little bit more clear!